<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>523</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1121</width>
    <height>639</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="logo_label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/Photo/Login/注意力增强.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>261</width>
     <height>46</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="LoginButton">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#LoginButton{
	font: 16pt &quot;Microsoft YaHei UI&quot;;	
	color: rgb(64, 83, 123);
}
QPushButton#LoginButton:hover{
	font: 16pt &quot;Microsoft YaHei UI&quot;;		
	color: rgb(112, 126, 155);
}</string>
      </property>
      <property name="text">
       <string>登录</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CancelButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#CancelButton{
	font: 16pt &quot;Microsoft YaHei UI&quot;;	
	color: rgb(64, 83, 123);
}
QPushButton#CancelButton:hover{
	font: 16pt &quot;Microsoft YaHei UI&quot;;		
	color: rgb(112, 126, 155);
}</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>270</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(39, 76, 94);</string>
      </property>
      <property name="text">
       <string>默认登陆</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Microsoft YaHei UI&quot;;
color: rgb(80, 193, 233);
color: rgb(39, 76, 94);</string>
      </property>
      <property name="text">
       <string>保存密码</string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>140</y>
     <width>401</width>
     <height>133</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="account_label">
          <property name="minimumSize">
           <size>
            <width>62</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>62</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(39, 76, 94);</string>
          </property>
          <property name="text">
           <string>账号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="user_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>YaHei Consolas Hybrid</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/Photo/Login/editline.png);
font: 14pt &quot;YaHei Consolas Hybrid&quot;;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>15</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="password_label">
          <property name="minimumSize">
           <size>
            <width>62</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>62</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>16</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 16pt &quot;Microsoft YaHei UI&quot;;
color: rgb(39, 76, 94);</string>
          </property>
          <property name="text">
           <string>密码</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwd_lineEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>31</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Adobe 仿宋 Std R</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">border-image: url(:/Photo/Login/editline.png);
font: 14pt &quot;Adobe 仿宋 Std R&quot;;
</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="LoginLabel">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="image_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Photo/Login/登陆界面_2.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="back_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>881</width>
     <height>531</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resource.qrc">:/Photo/Login/底色.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>back_label</zorder>
  <zorder>logo_label</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>image_label</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
